README.md

```diff
@@ -14,6 +14,12 @@ Changelog is kept at [docs/changelog.md](/docs/changelog.md).
 
 Check [examples](/docs/examples) dir for sample config files.
 
+## Setup
```

Comment:
I would clarify the environment this is for, this projects team is not on windows